src/autogluon/cloud/predictor/cloud_predictor.py

```diff
@@ -477,6 +477,7 @@ def fit(
             autogluon_sagemaker_estimator_kwargs["disable_profiler"] = True
             autogluon_sagemaker_estimator_kwargs["debugger_hook_config"] = False
         output_path = self.cloud_output_path + "/model"
+        code_location = self.cloud_output_path + "/code"
```

Comment:
Shall we use `os.path.join` instead?

Reply:
No. This is S3 path. `os.path.join` would result in `path\code` in Windows and would not work with S3

Reply:
Thanks for clarification